src/panels/lovelace/components/hui-entity-editor.ts

```diff
@@ -22,13 +22,13 @@ import type { HaEntityPicker } from "../../../components/entity/ha-entity-picker
 import "../../../components/ha-icon-button";
 import { sortableStyles } from "../../../resources/ha-sortable-style";
 import { HomeAssistant } from "../../../types";
-import { EntityConfig } from "../entity-rows/types";
+import { LovelaceRowConfig } from "../entity-rows/types";
 
 @customElement("hui-entity-editor")
 export class HuiEntityEditor extends LitElement {
   @property({ attribute: false }) protected hass?: HomeAssistant;
 
-  @property({ attribute: false }) protected entities?: EntityConfig[];
+  @property({ attribute: false }) protected entities?: LovelaceRowConfig[];
```

Comment:
This should not be in the entity-editor, as we use that for different cards, we should make a specific `entities-card-row-editor` that accepts a `LovelaceRowConfig[]`